### ISSUE TITLE
v0.1.3-alpha.1 fails to launch

### ISSUE THREAD

**Issue by richardwilkes** [Starfyre1/ACS#1]
Downloaded the .zip from GitHub, unpacked it, then tried to run it:

```
$ java -jar ACS.jar
Exception in thread "AWT-EventQueue-0" java.lang.ExceptionInInitializerError
	at com.starfyre1.startup.ACS$1.run(ACS.java:70)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:771)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:716)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:741)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: java.lang.NullPointerException: Cannot invoke "java.awt.Color.getRGB()" because "c" is null
	at java.desktop/javax.swing.plaf.ColorUIResource.<init>(ColorUIResource.java:85)
	at com.starfyre1.GUI.CharacterSheet.<clinit>(CharacterSheet.java:145)
	... 14 more```

**Comment by richardwilkes** [comment]
Add this to the run method in ACS:

```
				try {
					UIManager.setLookAndFeel(UIManager.getCrossPlatformLookAndFeelClassName());
				} catch (Exception ex) {
					System.err.println(ex);
				}
```

**Comment by Starfyre1** [comment]
Fix added and will be in v0.1.4-alpha.1 when it is created.
